src/Examine.Lucene/Suggest/FuzzySuggesterDefinition.cs

```diff
@@ -0,0 +1,75 @@
+using Lucene.Net.Analysis;
+using Lucene.Net.Index;
+using Lucene.Net.Search.Spell;
+using System.Linq;
+using Lucene.Net.Search.Suggest;
+using Lucene.Net.Search.Suggest.Analyzing;
+using Examine.Suggest;
+
+namespace Examine.Lucene.Suggest
+{
+    public class FuzzySuggesterDefinition : LuceneSuggesterDefinition, ILookupExecutor
+    {
+        public FuzzySuggesterDefinition(string name, string[] sourceFields = null, ISuggesterDirectoryFactory directoryFactory = null, Analyzer queryTimeAnalyzer = null)
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done